classes/Sensors/System.php

```diff
@@ -333,13 +333,15 @@ private function WriteLog($data, $ip, $username = '')
             $fp = $uploadsDirPath . $ip . $username . '.log';
 
             if (!$file = fopen($fp, 'a')) {
-                $i = 0;
+                $i = 1;
+                $fileOpened = false;
                 do {
                     $fp2 = substr($fp, 0, -4) . '_' . $i . '.log';
                     if ($file = fopen($fp2, 'a')) {
-                        exit;
+                        $fileOpened = true;
                     }
-                } while ($i > 0);
+                    $i++;
+                } while (!$fileOpened);
             }
```

Comment:
@carmelopreeostudios like this you're still not opening the file that was changed last.  if you have

- IP_USERNAME.log
- IP_USERNAME_1.log
- IP_USERNAME_2.log

you will open the first file if it's available.  We need to open _2.log if it's available and if not, create _3.log